client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -449,6 +449,14 @@ class SimplePaymentsDialog extends Component {
 							shouldDisplay={ this.returnTrue }
 						/>
 					}
+					secondaryAction={
+						<a
+							className="empty-content__action button"
+							href="https://support.wordpress.com/simple-payments/"
+						>
+							{ this.props.translate( 'Learn more about Simple Payments' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Learn more about Simple Payments.' )` *ES Score: 17*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/simple-payments-upgrade-nudge)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).